pkg/config/sink_protocol.go

```diff
@@ -38,6 +38,7 @@ const (
 	ProtocolCraft
 	ProtocolOpen
 	ProtocolCsv
+	ProtocolSimple
```

Comment:
Why use "Simple" as the new protocol name? We will add claim-check large message feature for this protocol, we will add water mark, checkpoint, schema tracking & bootstrap and more other features, can we use a proper name that can represent its functionalities? How about SufficientJson since it is json, but not each message contains the full schema information and is more sufficient than CanalJson?

Reply:
Or ExtensibleJson?

Reply:
The name is not decided yet, use `simple` temporarily.

`Json` should not be part of the protocol, since this new protocol does not only support JSON format, we may make it support other encoding formats such as avro, cbor.

Sufficient sounds good to me. Extensible is also a good name, but it's not easy to explain the `customized`, since this new protocol belongs to the TiCDC itself, all things is customized.

Reply:
For the concrete name, further discussion is required.